max_velocity_xy: 0.5 # m/s
max_velocity_z: 0.5 # m/s
control_rate_hz: 10.0 # Hz
takeoff_height: 3.0 # m
takeoff_wait_time: 10.0 # s
wire_yaw: 0.52359877559 # rad (30 degrees)
wire_x: 0.0 # m
wire_y: 0.0 # m
wire_z: 6.0 # m
